librosa/feature/spectral.py

```diff
@@ -1373,6 +1386,8 @@ def chroma_cqt(
 
     # Build the CQT if we don't have one already
     if C is None:
+        if y is None:
+            raise ParameterError("At least one of C or y must be provided to compute chroma")
```

Comment:
added some null checks here; they would have failed one level down the stack anyway, but the check really belongs here